oddbird/Metecho
# [storybook] deploy storybook to github pages

![](https://source.unsplash.com/featured/?cute,animal&stringlooop)






## Steps to test/reproduce
As of now  . . .
- `yarn run deploy-storybook` to deploy
- visit [this page](https://oddbird.github.io/Metecho) to see the published component library
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._


## Show me
_Provide screenshots/animated gifs/videos if necessary._
